load_helpers.py

```diff
@@ -0,0 +1,84 @@
+import openpyxl # library to read .xlsx format
+
+def checkHTMLTagClosures(row, rowCount):
+  tags_to_check = ["ol", "ul", "li", "a", "b"]
+  mismatches = {}
+  for key in row.keys():
+    if key == 'text' or key.startswith('text-'):
+      for tag in tags_to_check:
+        tag_opening_no_attr = "<" + tag + ">"
+        tag_opening_with_attr = "<" + tag + " "
+        tag_closing = "</" + tag + ">"
+        tag_openings = row[key].count(tag_opening_no_attr) + row[key].count(tag_opening_with_attr)
+        tag_closings = row[key].count(tag_closing)
+        if tag_openings > tag_closings:
+          print("WARNING: In row", str(rowCount), key, "has", str(tag_openings), "opening <" + tag + "> tag[s] but only", str(tag_closings), "closing tag[s].")
+
+
+def allRequiredFieldsPresent(requiredFields, optionalFields, row, rowCount):
```

Comment:
I don't think we ever actually use `requiredFields`, and instead this function assumes that any field not explicitly marked optional is required.  Unless I've missed something, let's remove that argument and the variable declarations altogether.